carrow.go

```diff
@@ -189,6 +189,51 @@ func (a *Array) Length() int {
 	return int(i)
 }
 
+// BoolAt returns bool at location
```

Comment:
This types at could be generated as well,
you only need to add a validation method to get_validate method (for time and string).